src/ceph-volume/ceph_volume/devices/lvm/prepare.py

```diff
@@ -256,30 +255,35 @@ def prepare(self):
             tags['ceph.block_device'] = block_lv.lv_path
             tags['ceph.block_uuid'] = block_lv.lv_uuid
             tags['ceph.cephx_lockbox_secret'] = cephx_lockbox_secret
-            tags['ceph.encrypted'] = encrypted
             tags['ceph.vdo'] = api.is_vdo(block_lv.lv_path)
 
+            def set_encryption_tag(tags, device_type):
+                tags.update({'ceph.encrypted': 1 if device_type in encrypt_devices else 0})
```

Comment:
```python
if device_type in encrypt_devices
```
this will fail when `--dmcrypt` is not passed as `self.args.dmcrypt` will be `None`